express-get-json/express-get-json-quiz-notes.md

```diff
@@ -0,0 +1,37 @@
+# express-get-json-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is an API endpoint?
+  The location from which APIs can access the resources they need to carry out their function.
+
+- What is the appropriate `Content-Type` header for HTTP messages that contain JSON in their bodies?
+  Res.json().
```

Comment:
`application/json` would be the appropriate `Content-Type` header for HTTP messages that contain JSON in their bodies.